theme-utils.mjs

```diff
@@ -704,7 +704,7 @@ async function versionBumpTheme(theme, addChanges) {
 	let styleCss = fs.readFileSync(`${theme}/style.css`, 'utf8');
 	let currentVersion = getThemeMetadata(styleCss, 'Version');
 
-	let filesToUpdate = await executeCommand(`find ${theme} -name package.json -o -name style.scss -o -name style-child-theme.scss -maxdepth 3`);
+	let filesToUpdate = await executeCommand(`find ${theme} -path "*/node_modules/*" -prune -name package.json -o -name style.scss -o -name style-child-theme.scss -maxdepth 3 -print`);
```

Comment:
It is ignoring `package.json` because it makes `and` condition with it

```suggestion
	let filesToUpdate = await executeCommand(`find ${theme} -not \\( -path "*/node_modules/*" -prune \\) -and \\( -name package.json -or -name style.scss -or -name style-child-theme.scss \\) -maxdepth 3`);
```